tests/callbacks/test_early_stopping.py

```diff
@@ -456,3 +456,38 @@ def test_early_stopping_squeezes():
         early_stopping._run_early_stopping_check(trainer)
 
     es_mock.assert_called_once_with(torch.tensor(0))
+
+
+@pytest.mark.parametrize("trainer", [Trainer(), None])
+@pytest.mark.parametrize(
+    "log_rank_zero_only, world_size, global_rank, expected_log",
+    [
+        (False, 1, 0, "bar"),
+        (False, 2, 0, "[rank: 0] bar"),
+        (False, 2, 1, "[rank: 1] bar"),
+        (True, 1, 0, "bar"),
+        (True, 2, 0, "[rank: 0] bar"),
+        (True, 2, 1, None),
+    ],
+)
+def test_early_stopping_log_info(tmpdir, trainer, log_rank_zero_only, world_size, global_rank, expected_log):
+    """checks if log.info() gets called with expected message when used within EarlyStopping."""
+
+    early_stopping = EarlyStopping(monitor="foo")
```

Comment:
Since `_log_info` is a staticmethod, you don't need to instantiate this object.

Reply:
Done